test/SlowTests/Server/Documents/PeriodicBackup/Azure.cs

```diff
@@ -21,6 +22,90 @@ public Azure(ITestOutputHelper output) : base(output)
         private const string AzureAccountName = "devstoreaccount1";
         private const string AzureAccountKey = "Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==";
 
+        [AzureStorageEmulatorFact(Skip = "Batch operations are not supported in emulator")]
```

Comment:
Please open an issue for that

Reply:
https://issues.hibernatingrhinos.com/issue/RavenDB-14306